File: serverless_framework/serverless_with_python3_and_parameter_store/handler.py
import json
import os


def hello(event, context):
    my_api_key = os.environ['MY_API_KEY']
    body = {
        "message": "MY_API_KEY is: {}".format(my_api_key),
        "input": event
    }
    response = {
        "statusCode": 200,
        "body": json.dumps(body)
    }

    return response

File: serverless_framework/serverless_with_global_var/handler.js
'use strict';

module.exports.hello = async event => {
  return {
    statusCode: 200,
    body: JSON.stringify(
      {
        message: (`Echo Environment Variable MYVAR is ${process.env.MYVAR}`),
        input: event,
      },
      null,
      2
    ),
  };

};
